### ISSUE TITLE
Functions instead of namespaces?

### ISSUE THREAD

**Issue by ScarylePoo** [taufik-nurrohman/markdown#3]
To preface, I'm not new to php. I'm not amazing by any means, but I'm not new to it. I am, however, inept when it comes to using namespaces.

My issue is this... I am writing my own website templating builder/engine. It's ofc FOSS, but I have it built around using md instead of html for pages. It's actually working out splendidly and my aim is to have something that even a novice developer could make a competent and nice-looking website fairly easily without the encumbrance of needing to use something like WordPress (in the past 12 or so years, most of my dev efforts have been geared towards wp, but I'm aiming for something fast, light, simple, and capable).

The repo for what I'm doing is here: https://github.com/ForbodingAngel/skeleton-website-template/tree/main

So basically, you can see the file structure. Everything is run through index.php thanks to htaccess

index calls 
config/config.php
then it calls the theme header, based upon variables set in config. In this case, the theme header is
themes/skeleton/header.php

Header then calls
```
<?php if ($loadplugins == true) { include './plugins/plugins.php'; } ?>
<?php include 'navigation.php' ?>
<?php include $layout; ?>
<?php include 'footer.php' ?>
```

$layout is gleaned by checking the md file that corresponds to the current uri, then the appropriate page layout is called.

So, I say all of this so that you can get an idea of how this all is set up. Currently, in config I have from/to as require, but the namespaces are killing me. I want to set the function alias in config, but it throws errors (Fatal error: Uncaught Error: Call to undefined function from_markdown() in F:\GitHub\skeleton-website-template\themes\skeleton\page.php...etc) about undefined functions unless I include the use function x\markdown\from as from_markdown; use function x\markdown\to as to_markdown; in the same file that I use echo from_markdown.

I have read the manual, I have searched for the answer, but this is a problem I do not know or understand how to solve. I am working around it for now, but my biggest goal with this project is to allow a novice -> intermediate web developer to create a site from more or less scratch in very little time with it containing all the bones to make exactly what you want without any bloat.

Can you help me? Hopefully this makes sense. I know it's a bit out there, I get it. Thank you <3 I really do love this parser, it works perfectly for my needs.

tl;dr I would like to be able to set the aliases for the functions somewhere in a single place at the beginning so that I can use from_markdown in any of the theme files, wherever I want.

**Comment by taufik-nurrohman** [comment]
Sure. If you cannot use alias, then you can return the namespace-ed function in your own global function:

~~~ php
<?php

require 'vendor/autoload.php';

function from_markdown(...$v) {
    return x\markdown\from(...$v);
}

function to_markdown(...$v) {
    return x\markdown\to(...$v);
}
~~~

**Comment by ScarylePoo** [comment]
That worked perfectly, thank you so much!
